util/random.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"time"
+	"math/rand"
+)
+
+// Get a random time duration between the lower bound and upper bound. This is useful because some of our automated tests
+// wound up flooding the AWS API all at once, leading to a "Subscriber limit exceeded" error.
+func GetRandomTime(lowerBound, upperBound time.Duration) time.Duration {
+	if lowerBound == upperBound {
+		return lowerBound
+	}
+
+	var lowerBoundMs, upperBoundMs float64
+
+	if upperBound < 1 * time.Second {
+		lowerBoundMs = 1
+		upperBoundMs = 1000
```

Comment:
It was an early implementation vestige. The thinking was that at a certain level, time ranges are so small as to not be meaningful, but no reason I need to enforce that. Will remove.